### PR TITLE
Achievement Section Done

### LINKED ISSUE #9: Achievement

### PR DESCRIPTION
**Pull request by dipikakoli26**
html and css code complete for achievement section
closes #9

### CODE REVIEW

**Comment by dipikakoli26** [comment]
added the achievement section code.
closes #9

**Comment by dipikakoli26** [comment]
Changes Done In Achievement Section.
closes #9

**Review by ajstyle2410 — APPROVED** [review]
done

**Comment by ajstyle2410** [comment]
ok

**Comment by dipikakoli26** [comment]
@ajstyle2410 Review My Changes I resolved Conficts.

File: index.html [hunk 1]
```diff
@@ -14,6 +14,17 @@
...
+            
+            <!-- Static List of Courses and Certificates -->
+            <ul id="coursesList">
+                <li> Java Full-Stack Web Developer - The Kiran Academy</li>
+                <li>Java Programming - Coursera</li>
+                <li>Full-Stack Web Development - Udemy</li>
+            </ul>
+        </section>
         <!-- Achievements Section -->
         <section class="achievements">
```
Comment (ajstyle2410) [inline index.html:29]: achievement section are not closed

**Comment by ajstyle2410** [comment]
done your code are accepted 👍

**Pull request merged**